spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -353,6 +359,21 @@ public SparkContext createSparkContext() {
       conf.set("spark.yarn.isPython", "true");
     }
 
+    // Update Settings for Mahout
+    String useMahout = getProperty("spark.mahout");
```

Comment:
this is redundent and I've deleted it. It will be reflected on my next push (docs, tests)